docs/license-handling.md

```diff
@@ -0,0 +1,41 @@
+# License handling
+
+ORT deals with various types of licenses. Their ORT-specific names and purposes are explained below.
+
+## Declared license
+
+The ORT *analyzer* picks up declared licenses that are provided as part of package manager specific metadata, like
+inside the `<licenses>` tag in the `pom.xml` for [Maven][1] or inside the `license` field in the `package.json` for
+[NPM][2]. In other words, this is the license the author of the package claims or intends the package to
+be licensed under; the license that is "visible from the outside". ORT does *not* consider licenses mentioned inside of
+source code files as declared licenses (see [detected licenses](#detected-licenses) below). As such, the declared
+license alone only provides an incomplete picture. There are so-called "envelope cases" where the license visible from
+the outside (*on* the envelope) does not match what is *inside* the envelope (i.e. in the source code). For example, a
+package might have declared itself to be licensed under the MIT license, but in the source code a file might contain a
+BSD-3-Clause license header.
+
+## Detected license
+
+Detected licenses are those licenses that are detected via an ORT *scanner* implementation by looking at the contents of
+all source code files belonging to a package, in particular at the contents of license files or copyright headers in
+source code files. Detected licenses complement the picture created by declared license by reveqaling envelope cases
+where the declared and detected licenses do not match.
+
+## Concluded license
+
+The concluded license is manually created via a [curation](config-file-curations-yml.md). In cases where the union of
+declared and detected licenses is wrong (e.g. due to mistakes in metadata or false positives from scanners), the
+concluded license is required to determine which licenses actually match reality. Curating a concluded license should be
+an objective decision based on verifiable facts.
+
+## Resolved license
+
+TODO
```

Comment:
@mnonnenmacher can you help me to find a wording here?

Reply:
That's actually difficult because the resolved license is not a property like the others, but an object that collects all available license information from package metadata, scan results, and configurations and provides functions to access this data and resolve the effective license. But maybe the second part of the previous sentence is the wording you are looking for ;) .

Reply:
Or should we simply drop it from this document, exactly because this is not a type of license the user is usually facing?

Reply:
Yes, I'll drop it for now.